ReactWindows/js/Components/Picker/PickerAndroid.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule PickerAndroid
+ */
+'use strict';
+
```

Comment:
Can you make sure this returns UnimplementedView, a la:
module.exports = require('UnimplementedView');

---
In reply to: [56820857](https://github.com/CatalystCode/react-native/pull/265#discussion_r56820857) [](ancestors = 56820857)